_component_design/checkboxes.md

```diff
@@ -17,150 +17,139 @@ last-modified: 2017-08-17
-    <div class="field">
-      <label>Select payment type</label>
-      <div class="ui buttons multi">
-        <button class="ui button ds-btn-med-sec active disabled">Credit / Debit</button>
-        <button class="ui button ds-btn-med-sec active disabled">Cash</button>
-        <button class="ui button ds-btn-med-sec disabled">Check</button>
-      </div>
-    </div>
-  </div>
-</div>
-
-## Error
-
-<div class="ui form error">
-  <div class="field error">
-    <label class="required">Select potential maintenance windows</label>
-    <div class="ui buttons multi">
-      <button class="ui button ds-btn-med-sec active error">12<span class="smCaps"> AM</span> &ndash; 4<span class="smCaps"> AM</span></button>
-      <button class="ui button ds-btn-med-sec active">4<span class="smCaps"> AM</span> &ndash; 8<span class="smCaps"> AM</span></button>
-      <button class="ui button ds-btn-med-sec">8<span class="smCaps"> AM</span> &ndash; 12<span class="smCaps"> PM</span></button>
-      <button class="ui button ds-btn-med-sec">12<span class="smCaps"> PM</span> &ndash; 4<span class="smCaps"> PM</span></button>
-      <button class="ui button ds-btn-med-sec">4<span class="smCaps"> PM</span> &ndash; 8<span class="smCaps"> PM</span></button>
-      <button class="ui button ds-btn-med-sec">8<span class="smCaps"> PM</span> &ndash; 12<span class="smCaps"> AM</span></button>
-    </div>
-    <div class="ui error message">
-      There is an error with the response to this question.
-    </div>
-  </div>
-</div>
+Checkboxes are used to change settings or bulk item selection. Checkboxes
+allow the user to select zero, one, or several items.
+
+{% figure [caption:"Default image"] [class:"image bg-light border"] %}
+ ![]({{site.url}}/assets/images/components/content-areas/checkboxes/checkbox-hero-image.svg){:width="80%"}
+ {% endfigure %}
+
+## When to use checkboxes
+
+* Using a checkable input to allows the user to select any number of options;
+  including zero, one, or several.
+* Use when selecting multiple items from a set, it is important for the
+  user to view all options at once. If viewing items side-by-side is not
+  important, consider a drop-down as it uses less space.
+* A stand-alone checkbox is used for a single option that the user can
+  turn on or off.
+* If there are multiple choices and only one option can be selected, a
+  radio button or drop-down list should be used.
+
+## Best practices for checkboxes
+
+* Each input is independent of all other input in the list and checking one
+  box does not uncheck the others.
+* A singular input may be used to enable or disable a feature, or it
+  may be used in place of two yes and no radio buttons.
+* Use checkboxes and radio buttons only to change settings, not as action
+  buttons that trigger another process or workflow.
+* In most cases, checkboxes default to having none of the options
+  selected. However, if Rackspace has a recommended choice, that may be
+  used as the default.
```

Comment:
Typically, checkboxes default to having no options selected. However, if Rackspace recommended an option, make that the default.